tests/test-scribe.c

```diff
@@ -254,46 +255,12 @@ fixture_set_up (Fixture *fixture,
   fixture->cancellable = g_cancellable_new ();
 }
 
-static void
-rm_r (const gchar *path)
-{
-  g_autoptr(GFile) file = g_file_new_for_path (path);
-  g_autoptr(GFileEnumerator) enumerator = NULL;
-  GFile *child = NULL;
-  GError *error = NULL;
-
-  enumerator = g_file_enumerate_children (
-      file, NULL, G_FILE_QUERY_INFO_NOFOLLOW_SYMLINKS, NULL, &error);
-  g_assert_no_error (error);
-
-  while (g_file_enumerator_iterate (enumerator, NULL, &child, NULL, &error) &&
-         child != NULL)
-    {
-      if (!g_file_delete (child, NULL, &error))
-        {
-          g_autofree gchar *child_path = g_file_get_path (child);
-          g_warning ("Failed to delete %s: %s", child_path, error->message);
-          g_clear_error (&error);
-        }
-    }
-
-  if (error != NULL)
-    {
-      g_warning ("Error while enumerating %s: %s", path, error->message);
-      g_clear_error (&error);
-    }
-
-  if (!g_file_delete (file, NULL, &error))
-    {
-      g_warning ("Failed to delete %s: %s", path, error->message);
-      g_clear_error (&error);
-    }
-}
-
 static void
 fixture_tear_down (Fixture *fixture,
                    gconstpointer user_data)
 {
+  g_autoptr(GError) error = NULL;
```

Comment:
Nitpick: As I think I mentioned on one of the earlier comments, the approach I’ve been taking recently has been to consistently call local `GError*`s `local_error`, to distinguish them from a possible `GError **error` parameter.